src/store/modules/user/actions.ts

```diff
@@ -59,6 +59,7 @@ const actions: ActionTree<UserState, RootState> = {
         emitter.emit('timeZoneDifferent', { profileTimeZone: resp.data.userTimeZone, localTimeZone});
       }
       commit(types.USER_INFO_UPDATED, resp.data);
+      commit(types.USER_CURRENT_FACILITY_UPDATED, resp.data.facilities.length > 0 ? resp.data.facilities[0] : {});
```

Comment:
I think this is not related and something left off from the previous work. Please add information about it in PR description as Additional Change